src/scala/com/github/johnynek/bazel_deps/Resolver.scala

```diff
@@ -66,7 +66,12 @@ class Resolver(servers: List[MavenServer], resolverCachePath: Path) {
       val exclusion = new Exclusion(elem.group.asString, elem.artifact.asString, "", "jar")
       exclusions.add(exclusion)
     }
-    collectRequest.setRoot(new Dependency(new DefaultArtifact(m.asString), "", false, exclusions))
+    val classifier = m.classifier match {
+      case Classifier("") => null
+      case Classifier(c) => c
+    }
+    collectRequest.setRoot(new Dependency(new DefaultArtifact(
+      m.group.asString, m.artifact.asString, classifier, null, m.version.asString), "", false, exclusions))
```

Comment:
can you comment what the `null` and `false` parameters are here so people can see without looking up the docs on Aether?

Reply:
Yep, will do.